package/src/utils/define-integration.ts

```diff
@@ -34,9 +34,7 @@ export const defineIntegration = <
 	setup,
 }: {
 	name: string;
-	defaults: {
-		[Property in keyof TOptions]-?: TOptions[Property];
-	};
+	defaults?: Required<TOptions>;
```

Comment:
```suggestion
	defaults?: Required<TOptions>;
```
Then it may required updating some types when using defu + adding stuff to the docs: integration authors must be strict, like use `prop?: string | undefined` instead of `prop?: string`